internal/expr/parser_helper_test.go

```diff
@@ -132,3 +132,47 @@ func (s *ExprInternalSuite) TestUnfinishedQuote(c *C) {
 		}
 	}
 }
+
+func (s *ExprInternalSuite) TestValidComments(c *C) {
+	var p = NewParser()
+
+	validComments := []string{
+		`-- Simple comment`,
+		`-- Simple with line break
+		`,
+		`/* muti
+		 line */`,
+		`/* unfinished mutiline`,
+		`/* -- */`,
+		`-- */`,
```

Comment:
Perhaps it would be good to also test `"--"` and `"/*"`

Reply:
Added the test cases